pkg/server/plugin/datastore/datastore.go

```diff
@@ -14,9 +14,9 @@ type DataStore interface {
 	// Bundles
 	AppendBundle(context.Context, *AppendBundleRequest) (*AppendBundleResponse, error)
 	CountBundles(context.Context) (int32, error)
-	CreateBundle(context.Context, *CreateBundleRequest) (*CreateBundleResponse, error)
-	DeleteBundle(context.Context, *DeleteBundleRequest) (*DeleteBundleResponse, error)
-	FetchBundle(context.Context, *FetchBundleRequest) (*FetchBundleResponse, error)
+	CreateBundle(context.Context, *common.Bundle) (*common.Bundle, error)
+	DeleteBundle(context.Context, string, DeleteMode) error
+	FetchBundle(context.Context, string) (*common.Bundle, error)
```

Comment:
nit: it would be nice to have the variable named here (e.g., trustDomainID), `string` isn't self documenting enough.

Reply:
will do